zappa/handler.py

```diff
@@ -260,9 +260,9 @@ def _process_exception(cls, exception_handler, event, context, exception):
             try:
                 handler_function = cls.import_module_and_get_function(exception_handler)
                 exception_processed = handler_function(exception, event, context)
-            except Exception as cex:
-                logger.error(msg="Failed to process exception via custom handler.")
-                print(cex)
+            except Exception:
+                logger.exception("Failed to process exception via custom handler.")
```

Comment:
output the exception and output the error message.

```
...
except Exception as e:
    logger.exception(e)
    logger.error("Failed to process exception via custom handler.")
```

Reply:
This is usually unnecessary. logger.exception() logs the exception, too, and gives you a traceback after the error message.